locales/es/translation.json

```diff
@@ -1,1327 +1,1328 @@
+    "Edit Project": "Modificar proyecto",
+    "Edit repository": "Editar repositorio",
+    "Edit Role": "Editar rol",
+    "Edit Rule": "Editar regla",
+    "Edit team": "Editar equipo",
+    "Edit Team": "Editar equipo",
+    "Edit template": "Editar plantilla",
+    "Edit user": "Editar el usuario",
+    "Edit User": "Modificar usuario",
+    "Edit workflow template": "Editar plantilla de flujo de trabajo",
+    "Elapsed": "Tiempo transcurrido",
+    "Email": "Correo electrónico",
+    "Enable content signing to verify that the content has remained secure when a project is synced. If the content has been tampered with, the job will not run.": "Activa la firma de contenido para verificar que el contenido permanece seguro cuando se sincroniza un proyecto. Si el contenido ha sido manipulado, el trabajo no se ejecutará.",
+    "Enable fact storage": "Habilitar el almacenamiento de datos",
+    "Enable rulebook activations_one": "Habilitar activaciones del reglamento",
+    "Enable rulebook activations_other": "Habilitar activaciones del reglamento",
+    "Enable rulebookActivations_one": "Habilitar activaciones del reglamento",
+    "Enable rulebookActivations_other": "Habilitar activaciones del reglamento",
+    "Enable selected activations": "Habilitar activaciones seleccionadas",
+    "Enable webhook": "Activar Webhook",
+    "Enabled": "Habilitado",
+    "Enabled options": "Opciones habilitadas",
+    "Encrypted": "Cifrado",
+    "End date": "Fecha de terminación",
+    "Enter a friendly name for the automation server": "Introduzca un nombre descriptivo para el servidor de automatización",
+    "Enter collection": "Introducir colección",
+    "Enter controller token": "Introducir token de controlador",
+    "Enter credential token": "Introducir token de la credencial",
+    "Enter description": "Introducir descripción",
+    "Enter description ": "Introducir descripción ",
+    "Enter email": "Introducir correo electrónico",
+    "Enter first name": "Introducir nombre",
+    "Enter image name": "Introducir nombre de la imagen",
+    "Enter inventory": "Introducir inventario",
+    "Enter job template": "Introducir plantilla de trabajo",
+    "Enter keywords": "Introducir palabras clave",
+    "Enter last name": "Introducir apellido",
+    "Enter name": "Introducir nombre",
+    "Enter namespace": "Introducir espacio de nombres",
+    "Enter organization": "Introducir organización",
+    "Enter password": "Introducir contraseña",
+    "Enter SCM URL": "Introducir URL del SCM",
+    "Enter tags": "Introducir etiquetas",
+    "Enter the time it takes to run the following templates manually.": "Introduzca el tiempo que se tarda en ejecutar manualmente las siguientes plantillas.",
+    "Enter the url of the automation server": "Introducir url del servidor de automatización",
+    "Enter username": "Introducir nombre de usuario",
+    "Enter workflow job template": "Introducir plantilla de tarea de flujo de trabajo",
```

Comment:
In the rest of places "workflow job template" is translated as "plantilla de trabajo del flujo de trabajo"